### PR TITLE
RDM-9258: Release RDM-8842 with CI/CD

### PR DESCRIPTION
**Pull request by mattnayler**
### JIRA link (if applicable) ###

   [RDM-9258](https://tools.hmcts.net/jira/browse/RDM-9258) _"Release RDM-8842 with CI/CD"_
   [RDM-8842](https://tools.hmcts.net/jira/browse/RDM-8842) _"Increment Assigned User Count of Cases when Assigned to Users"_

### Change description ###

Release RDM-8842 with CI/CD

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 1]
```diff
@@ -15,8 +15,8 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-224.service.core-compute-preview.internal/
+    #DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
```
Comment (smathangi) [inline charts/ccd-data-store-api/values.preview.template.yaml:18]: Any reason to comment these? Do we want o keep aat urls on master branch for preview?

**Review by smathangi — APPROVED** [review]

File: charts/ccd-data-store-api/values.preview.template.yaml (hunk 1 above)
Comment (MSancaktutar) [reply charts/ccd-data-store-api/values.preview.template.yaml:18]: Yes, we need to comment these to be able to wire the pods in preview with ccd dependencies on aat.
We will change it back as soon as we merge back to develop from master, after release.

Comment (MSancaktutar) [reply charts/ccd-data-store-api/values.preview.template.yaml:18]: The pipeline improvement is taking care of wiring for FTAs on running on Jenkins agents, not those of CI/CD releasables on preview. That part requires further work to make more practically manageable.

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request closed without merging**